smaps/src/test/java/com/google/smaps/HistogramTest.java

```diff
@@ -32,53 +34,67 @@
 import org.mockito.MockitoAnnotations;
 
 /**
- * Unit tests for {@link Homepage}.
+ * Unit tests for {@link Histogram}.
  */
 @RunWith(JUnit4.class)
-public class HomepageTest {
-  private static final String FAKE_URL = "fake.fk/home";
-  // Set up a helper so that the ApiProxy returns a valid environment for local testing.
+public class HistogramTest {
+  private static final String FAKE_URL = "fake.fk/histogram";
   private final LocalServiceTestHelper helper = new LocalServiceTestHelper();
 
   @Mock private HttpServletRequest mockRequest;
   @Mock private HttpServletResponse mockResponse;
   private StringWriter responseWriter;
-  private Homepage servletUnderTest;
+  private Histogram servletUnderTest;
 
   @Before
   public void setUp() throws Exception {
+    // Sets up the servlet and mock request and response.
     MockitoAnnotations.initMocks(this);
     helper.setUp();
 
-    //  Set up some fake HTTP requests
+    //  Sets up some fake HTTP requests
     when(mockRequest.getRequestURI()).thenReturn(FAKE_URL);
 
-    // Set up a fake HTTP response.
+    // Sets up a fake HTTP response.
     responseWriter = new StringWriter();
     when(mockResponse.getWriter()).thenReturn(new PrintWriter(responseWriter));
 
-    servletUnderTest = new Homepage();
+    servletUnderTest = new Histogram();
   }
 
   @After
   public void tearDown() {
+    // Tears down the Servlet after tests are done.
     helper.tearDown();
   }
 
   @Test
-  public void doGet_writesResponse() throws Exception {
+  public void doGetWritesJson() throws Exception {
```

Comment:
You can write this as doGet_writesJson as per https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names go for it. I'm okay either way, but I do tend to like the underscores.